src/BlockArea.h

```diff
@@ -362,7 +362,7 @@ class cBlockArea
 	bool HasBlockMetas    (void) const { return (m_BlockMetas    != nullptr); }
 	bool HasBlockLights   (void) const { return (m_BlockLight    != nullptr); }
 	bool HasBlockSkyLights(void) const { return (m_BlockSkyLight != nullptr); }
-	bool HasBlockEntities (void) const { return (m_BlockEntities != nullptr); }
+	bool HasBlockEntities (void) const { return m_BlockEntities.operator bool(); }
```

Comment:
:O what's the reason here?

Reply:
You mean for the weird access to the book operator?

Normally you can use it simply. I don't know why the compiler denied it here